projects/packages/publicize/src/class-publicize-ui.php

```diff
@@ -592,38 +588,23 @@ private function get_metabox_form_connected( $connections_data ) {
 		<?php
 
 		foreach ( $connections_data as $connection_data ) {
-			$all_connections_done = $all_connections_done && $connection_data['done'];
-			$connection_healthy   = ! isset( $connection_data['is_healthy'] ) || $connection_data['is_healthy'];
-			if ( ! $connection_healthy ) {
-				$broken_connections[] = $connection_data;
-
-			}
 			?>
 
 			<li>
 				<label
-					for="wpas-submit-<?php echo esc_attr( $connection_data['id'] ); ?>"
+					for="wpas-submit-<?php echo esc_attr( $connection_data['connection_id'] ); ?>"
 				>
 					<input
 						type="checkbox"
-						name="wpas[submit][<?php echo esc_attr( $connection_data['id'] ); ?>]"
-						id="wpas-submit-<?php echo esc_attr( $connection_data['id'] ); ?>"
+						name="wpas[submit][<?php echo esc_attr( $connection_data['connection_id'] ); ?>]"
+						id="wpas-submit-<?php echo esc_attr( $connection_data['connection_id'] ); ?>"
 						class="wpas-submit-<?php echo esc_attr( $connection_data['service_name'] ); ?>"
 						value="1"
-						data-id="<?php echo esc_attr( $connection_data['id'] ); ?>"
+						data-id="<?php echo esc_attr( $connection_data['connection_id'] ); ?>"
 					<?php
-						checked( true, $connection_data['enabled'] && $connection_healthy );
-						disabled( false, $connection_healthy );
+						checked( true, $connection_data['enabled'] );
 					?>
 					/>
-				<?php if ( $connection_data['enabled'] && $connection_healthy && ! $connection_data['toggleable'] ) : // Need to submit a value to force a global connection to POST. ?>
-					<input
-						type="hidden"
-						name="wpas[submit][<?php echo esc_attr( $connection_data['id'] ); ?>]"
-						value="1"
-					/>
-				<?php endif; ?>
-
```

Comment:
This is strange! Perhaps there used to a function which meant that shared connections couldn't be disabled by authors/editors?